services/ui-src/src/components/tables/SortableTable.tsx

```diff
@@ -57,6 +61,15 @@ export const SortableTable = ({
     },
   });
 
+  useEffect(() => {
+    const relKeys = Object.keys(headerRefs.current);
+    const refLabels = relKeys.reduce((obj, key) => {
+      obj[key] = headerRefs.current[key].textContent || "";
```

Comment:
if multiple items end up getting the or condition of "", that would cause a problem, right?